SUSE/rmt
# Add release_stage attribute to ProductSerializer

## Description
**See:** https://bugzilla.suse.com/show_bug.cgi?id=1176628
**Part of:** https://trello.com/c/Q5zEBJvj/1996-project-maintenance-connect-rmt-yast-rmt

Add `release_stage` to `V3::ProductSerializer` to allow `yast` and other programs correctly identify released products.

## How to test
- Run this branch on your local machine
- Call any products endpoint and check `release_stage` is included in the `json` response. (https://scc.suse.com/connect/v4/documentation)

## Change Type

*Please select the correct option.*


## Checklist

*Please check off each item if the requirement is met.*


## Other Notes

Please use this space to provide notes or thoughts to the team, such as tips on how to review/demo your changes.
